### ISSUE TITLE
[GMOD] Props seem to magnetize to the player

### ISSUE THREAD

**Issue by ScottVordem** [Joshua-Ashton/VPhysics-Jolt#88]
I noticed that when I approach a light object (like a small wooden box), it starts tumbling against me pretty fast (can kill the player), and if I back away slowly, it follows me. Very strange behavior.

**Comment by klen-list** [comment]
sounds like #18

**Comment by ScottVordem** [comment]
Yep, Increasing `vjolt_player_collision_tolerance` amplifies the effect, you can have props hurtle towards you from across the map if high enough.

But like in the other issue, setting `vjolt_player_collision_tolerance` to `0` fixes it.
